README.md

```diff
@@ -1,10 +1,12 @@
 # Puppet module for Corosync & Pacemaker
 
-[![Puppet Forge Release](https://img.shields.io/puppetforge/v/puppet/corosync.svg)](https://forge.puppet.com/puppet/corosync)
-[![Downloads](https://img.shields.io/puppetforge/dt/puppet/corosync.svg)](https://forge.puppet.com/puppet/corosync) 
-[![Build Status](https://travis-ci.org/voxpupuli/puppet-corosync.png?branch=master)](https://travis-ci.org/voxpupuli/puppet-corosync)
-[![Endorsement](https://img.shields.io/puppetforge/e/puppet/corosync.svg)](https://forge.puppet.com/approved)
-[![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/533/badge)](https://bestpractices.coreinfrastructure.org/projects/533)
```

Comment:
Do you not like this one?

Reply:
I can add it back, I didn't see it in our other ones, and thought we were trying to keep a standard set? Re-added.
Also the scores one doesn't seem to be working AFAICT - I get https://img.shields.io/puppetforge/f/puppet/corosync.svg 'score unknown' for this despite the forge having a score (same for mongodb)

Reply:
yes the CII badge is proving the quality of this repo. We should instead add it to other repositories.